### DISCUSSION TITLE
Source code for antlr4-4.8-2-Rust0.3.0-beta

### DISCUSSION THREAD

**Discussion by yuriry** [rrevenantt/antlr4rust#63] [Q&A] [answered]
Where can I find source code for [antlr4-4.8-2-Rust0.3.0-beta](https://github.com/rrevenantt/antlr4rust/releases/tag/antlr4-4.8-2-Rust0.3.0-beta)?

I'm trying to solve [the problem with partially implemented Listenable](https://github.com/rrevenantt/antlr4rust/issues/61), but when I build [rust-target](https://github.com/rrevenantt/antlr4/tree/rust-target) branch, the generated code seems old - it does not compile.

**Comment by rrevenantt** [comment, marked as answer]
I think your issue was because of incorrect build instructions in README, its fixed by #57 which i have just merged.

Also i have already started working on that #61 issue.

**Reply by yuriry** [reply]
Thank you so much, really appreciate your response!  Adding `--remote` to `submodule update` fixed the problem.
